contracts/MyStrategy.sol

```diff
@@ -205,7 +216,7 @@ contract MyStrategy is BaseStrategy {
         uint256 _before = IERC20Upgradeable(want).balanceOf(address(this));
 
         // figure out and claim our rewards
-        ICurveGauge(gauge).claim_rewards();
+        ICurveGaugeFactory(gaugeFactory).mint(gauge);
```

Comment:
The GaugeFactory from the test doesn't contain a .mint() function: https://etherscan.io/address/0xabC000d88f23Bb45525E447528DBF656A9D55bf5#writeContract

![image](https://user-images.githubusercontent.com/25463788/169118276-5920869b-7ad2-403b-8820-e1d7e13c1051.png)

Reply:
Isn't it supposed to be `transmit_emissions`?

Reply:
It seems like the address on the test points to the rootGaugeFactory and we need the childGaugeFactory

Reply:
Nevermind, sorry, was looking at Etherscan... The same address on Arbiscan is the ChildGaugeFactory indeed: https://arbiscan.io/address/0xabC000d88f23Bb45525E447528DBF656A9D55bf5, very weird...

Reply:
Confirm that `factory.mint(gauge)` indeed mints the amount accumulated by the user (strat) on the gauge in matter and transfers the amount directly to the user: https://github.com/curvefi/curve-xchain-factory/blob/6066c873713c50c6baaacad460b7be48fbd45846/contracts/ChildGaugeFactory.vy#L107-L113

Not very intuitive. Very good finding!

Reply:
arbiscan